### DISCUSSION TITLE
【问题】如何更快速的进行开发调试

### DISCUSSION THREAD

**Discussion by 1741718728** [win12-online/win12#330] [Q&A] [answered]
你们好我有一个疑问，请问一下你们是怎么开发调试的呀？比如说改动了一个样式，我想看看效果，刷新浏览器，系统就会回到登录页，这样开发特别费时间

**Comment by User782Tec** [comment, marked as answer]
噢我们一般有两种解决方案
第一是可以用vscode的live server插件，启动该服务之后样式变动就不用刷新页面了，但要注意可能会触发缓存机制导致更新不及时，可参考下面方案
第二是可以在开发用的网址（可以是localhost, IP或者域名，file:///应该也行）后面加上"?develop=true"（不包括双引号），就能在刷新之后自动跳过登录界面和开机弹窗，并禁用缓存机制

**Comment by NB-Group** [comment]
行

**Comment by 1741718728** [comment]
好的感谢解答，我按照这种方式试一下

**Comment by ghost** [comment]
AI Copilot 也能吗？@User782Tec

**Reply by NB-Group** [reply]
斯，不理解你的意思。
AI Copilot的代码都挂在我的cloudflare上，转发代码挂在我的服务器上，你怎么调试？

**Reply by NB-Group** [reply]
不懂的话看一下[我写的这个](https://github.com/tjy-gitnub/win12/blob/main/scripts/AI%20Copilot%20service/README.md)，里面详细介绍了AI的架构

**Reply by Mbilse** [reply]
> 斯，不理解你的意思。
> AI Copilot的代码都挂在我的cloudflare上，转发代码挂在我的服务器上，你怎么调试？
> 

不是，你们的cloudflare 帐户打开时就没卡过吗，我之前的几个cloudflare 帐户进入dashboard 之后就不加载任何页面了

**Comment by tjy-gitnub** [comment]
可以用edge的开发工具 >u-)o
用edge从本地打开html，在devtool里面按提示设置工作区根目录，edge会实时同步css文件的更改（仅限单独的css文件），挺好用的
![image](https://github.com/tjy-gitnub/win12/assets/71509955/7964aed7-b993-426e-b21e-3b7fed9a53e5)
